hw_diag/tasks.py

```diff
@@ -48,11 +49,15 @@ def perform_hw_diagnostics(ship=False):  # noqa: C901
         log.exception(e)
 
     # Get the blockchain height from the Helium API
-    value = "1"
+    value = "Not Available"
```

Comment:
Can you make sure these changes are compatible with the diagnostics [uploads schema](https://github.com/NebraLtd/hm-diag/blob/adc65aaed9b898313a9259c9ec0af9b566f228cf/bigquery/bq_schema.json#L111).

Reply:
This is not compatible, I'll set it back to 1. I'll have to see how the other values are affected referring to the big query schema and I may have to make changes to the view to accommodate for displaying a string instead